field_history/migrations/0001_initial.py

```diff
@@ -12,7 +12,6 @@ class Migration(migrations.Migration):
     initial = True
 
     dependencies = [
-        ('contenttypes', '0002_remove_content_type_name'),
```

Comment:
The `FieldHistory` model has a `ContentType` field, so we'll need to make sure that the migrations for `contenttypes` have completed before creating the `FieldHistory` table or else this migration will fail.